typescript/nextcloud-with-grafana/README.md

```diff
@@ -0,0 +1,34 @@
+This example creates
+- nextcloud
+- mariadb
+- redis
+- traefik
+- cert-manager
+- certificate issuer
+- victoria metrics
+- grafana
+## Config
+you can config project using `yamls/config.yaml` and these values at the top of `main.ts`
```

Comment:
Could you also briefly mention what `secrets.yaml` and `victoria.yaml` do in this project?

Reply:
I changed secerts.yaml to database_credentials.yaml and mentioned them in README.md
```
Database credentials are in `yamls/database_credentials.yaml`  
Victoria metrics config file is in `yamls/victorial.yaml`, this file contains scrape configs.
```